addons/website/static/src/js/content/menu.js

```diff
@@ -660,6 +657,130 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
     },
 });
 
+publicWidget.registry.megaMenuDropdown = publicWidget.Widget.extend({
+    selector: "header#top",
+    disabledInEditableMode: false,
```

Comment:
Yes I made a specific widget so it is easier to understand what it does (like for the `hoverableDropdown`) since it is very specific to the mega menus :slightly_smiling_face: 

I didn't know for the capitalized, I will change it, thanks